packages/thalaswap-router/src/abi/coin_wrapper.ts

```diff
@@ -1,8 +1,35 @@
 export const COIN_WRAPPER_ABI = {
-  address: "0x8a08715c6dc0a402c89c446b8ba75fae867f040247b526d309f91a76f098a52e",
+  address: "0x47078bc1e50f803987fbbf69b706cebb29ea4a54dac8b0ce2d9e7af90a41d02f",
```

Comment:
guess this is on testnet? then we probably do not want to merge in yet? or it doesn't matter